backend-issuer/src/controllers/webhooks/index.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  Controller,
+  Body,
+  Post,
+  Route,
+  SuccessResponse,
+  Tags,
+  Hidden,
+} from 'tsoa'
+import { injectable } from 'tsyringe'
+
+import { logger } from '../../lib/logger.js'
+import { InternalError } from '../../lib/error-handler/index.js'
+import Database from '../../lib/db.js'
+
+@Route('api/webhooks')
+@Tags('webhooks')
+@injectable()
+export class WebhooksController extends Controller {
+  constructor(private db: Database) {
+    super()
+  }
+
+  /**
+   * @summary Confirm membership with received verification code
```

Comment:
looks like the summaries in this file are copied from somewhere else, not that it matters too much since they're hidden routes...

Reply:
good catch, fixed